### ISSUE TITLE
Glyphicons font is not loaded

### ISSUE THREAD

**Issue by gino0631** [adminfaces/admin-theme#255]
##### Issue Overview

Glyphicons font is not loaded starting from version 1.3.1, and the icons using that font are not displayed correctly.

##### Current Behaviour

The following errors are displayed in browser console:
```
theme.css.xhtml:1          GET http://127.0.0.1:8080/webapp/fonts/glyphicons-halflings-regular.woff2 net::ERR_ABORTED 404
theme.css.xhtml:1          GET http://127.0.0.1:8080/webapp/fonts/glyphicons-halflings-regular.woff net::ERR_ABORTED 404
theme.css.xhtml:1          GET http://127.0.0.1:8080/webapp/fonts/glyphicons-halflings-regular.ttf 404
```

And the triangle to collapse table rows is not displayed correctly:
![image](https://user-images.githubusercontent.com/13659309/155861741-bd2e63e7-5820-466d-a027-4241fc9153c0.png)

##### Expected Behaviour

As in version 1.3.0, the fonts should be loaded from resources, e.g.:
```
http://127.0.0.1:8080/webapp/javax.faces.resource/fonts/glyphicons-halflings-regular.woff2.xhtml?ln=bootstrap
```

And the triangle to collapse rows should be displayed correctly:
![image](https://user-images.githubusercontent.com/13659309/155861898-c98b59dd-5aa5-4b31-8d3e-612dc8236cae.png)

##### How to reproduce  

A data table with an expandable header row will show the visual impact of the problem:

```html
  <p:dataTable ...>
    <p:headerRow expandable="true" ...>
    </p:headerRow>
  </p:dataTable>
```

##### Possible causes

In 5fc733d, font URLs in bootstrap.css were (most probably unintentionally) changed to contain relative URLs instead of resource references:
```
src: url("../fonts/glyphicons-halflings-regular.eot");
```

##### Additional Information

* AdminFaces version: 1.3.2
* PrimeFaces version: 10.0.0
* Browser version: Google Chrome Version 98.0.4758.102 (Official Build) (64-bit)

**Comment by rmpestano** [comment]
Hey, true thanks for the feedback! I'll fix asap

**Comment by rmpestano** [comment]
Hi again @gino0631, can you try `1.3.3-SNAPSHOT`? you'll need to [enable snapshots](https://github.com/adminfaces/admin-theme#9-snapshot) to use that version

**Comment by gino0631** [comment]
@rmpestano thanks, now it works. I encountered just one minor issue: `admin-template:1.3.3-SNAPSHOT` still depends on `admin-theme:1.3.2`, so I had to add `admin-theme:1.3.3-SNAPSHOT` to dependency management explicitly (normally I have a dependency on `admin-template` only).

**Comment by rmpestano** [comment]
Hey @gino0631, this fix is available in `v1.4.0`, thanks for the feedback!
